packages/fxa-customs-server/lib/email_record.js

```diff
@@ -273,14 +273,14 @@ module.exports = function (limits, now) {
           (otpVerificationTime) =>
             otpVerificationTime >
             thisIsNow - limits.passwordResetOtpVerificationRateLimitWindowMs
-        ).length > limits.maxPasswordResetOtpVerificationRateLimit
+        ).length >= limits.maxPasswordResetOtpVerificationRateLimit
```

Comment:
Should this call `this.trimPasswordResetOtpVerifications(now());` before calling the function in return?

Reply:
Not really necessary since `isOverPasswordResetOtpVerificationBlockLimit` is going to call it, and we check both limits per action.